webClient/src/app/app.component.ts

```diff
@@ -431,6 +431,20 @@ export class AppComponent implements AfterViewInit {
   loadZssSettings(): Observable<ZssConfig> {
     return this.http.get(ZoweZLUX.uriBroker.serverRootUri("server/proxies")).map((res: Response) => res.json());
   }
+
+  saveSettings() {
+    this.http.put(ZoweZLUX.uriBroker.pluginConfigForScopeUri(this.pluginDefinition.getBasePlugin(), 'user', 'sessions', '_defaultTN3270.json'),
```

Comment:
This doesn't work for me and I think I know why: angular does not let you do an http request unless you subscribe to the response. It is a weird requirement. So, you can `.subscribe(){}` and actually not read the response, or do a `this.log.debug('Saved config')` inside, but you must subscribe.

Reply:
Thanks!  I guess I have gaps in my angular knowledge.  All these different frameworks like react, angular solve the same problem in different ways.  They are also similar to android. So whenever I try to read them I tend to not give importance on specific details on a per framework basis but on how to solve the problem. 
I generally find the starting point of the app and put a bunch of printf statements and explore the code till I find the problem area. So till now I have not tried to memorize  functions. I go to the docs and read the main/basic components section so that I know enough to solve problems.  I don't know if this is the prefect way of solving problems or if there is any other better way.  Your suggestion would help me greatly!